src/page-weekly-snapshot/WeeklySnapshotPage.tsx

```diff
@@ -0,0 +1,195 @@
+import { maxBy, minBy, orderBy } from "lodash";
+import numeral from "numeral";
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+
+import { DateMMMMdyyyy } from "../design-system/DateFormats";
+import InputSelect from "../design-system/InputSelect";
+import Loading from "../design-system/Loading";
+import { Column } from "../design-system/PageColumn";
+import {
+  LocaleDataProvider,
+  LocaleRecord,
+  useLocaleDataState,
+} from "../locale-data-context";
+import {
+  NYTCountyRecord,
+  NYTData,
+  NYTStateRecord,
+  useNYTData,
+} from "./NYTDataProvider";
+
+const stateNamesFilter = (key: string) =>
+  !["US Total", "US Federal Prisons"].includes(key);
+const formatNumber = (number: number) => numeral(number).format("0,0");
+
+const LocaleStats = styled.div`
+  display: flex;
+`;
+const LocaleStatsList = styled.ul``;
+
+type PerCapitaCountyCase = {
+  name: string;
+  casesIncreasePerCapita: number | undefined;
+};
+
+function getDay(nytData: NYTCountyRecord[] | NYTStateRecord[], day: number) {
+  if (day === 1) {
+    return minBy(nytData, (d: NYTCountyRecord | NYTStateRecord) => d.date);
+  } else {
+    return maxBy(nytData, (d: NYTCountyRecord | NYTStateRecord) => d.date);
+  }
+}
+
+function getCountyDataByName(counties: NYTCountyRecord[]) {
+  const dataByCounty: { [countyName: string]: NYTCountyRecord[] } = {};
+
+  counties.forEach((data: NYTCountyRecord) => {
+    if (dataByCounty[data.county]) {
+      dataByCounty[data.county] = [...dataByCounty[data.county], data];
+    } else {
+      dataByCounty[data.county] = [data];
+    }
+  });
+
+  return dataByCounty;
+}
+
+function calculatePerCapitaIncrease(
+  daySevenCases: number | undefined,
+  dayOneCases: number | undefined,
+  countyPopulation: number | undefined,
+) {
+  if (
+    (!daySevenCases && daySevenCases !== 0) ||
+    (!dayOneCases && dayOneCases !== 0) ||
+    !countyPopulation
+  )
+    return;
+  return (daySevenCases - dayOneCases) / countyPopulation;
```

Comment:
Just want to shine a light and confirm this is in fact the right calculation for "per capita increase in cases" and that there's no additional need to, for example, multiply by 100 to get to the right percentage. I don't have any explicit reason to think this is not correct, but it's the kind of thing that can be tricky so want to be sure.

Reply:
It's not super clear but this number does get multiplied to 100 before displaying it on [Line 136](https://github.com/Recidiviz/covid19-dashboard/pull/562/files#diff-117efcb95f817f083faeb4c1c790f02aR136) with the [`numeral().format("0.000 %")`](https://github.com/adamwdraper/Numeral-js/blob/master/src/formats/percentage.js) formatter. If that's too hidden I could multiply it more explicitly and then format it with numeral without using their percentage formatter.